packages/airnode-node/src/evm/requests/event-logs.ts

```diff
@@ -33,12 +34,13 @@ export function parseAirnodeRrpLog<T = { readonly args: any }>(log: ethers.provi
 }
 
 export async function fetch(options: FetchOptions): Promise<EVMEventLog[]> {
-  // Protect against a potential negative fromBlock value
+  // Protect against a potential negative fromBlock and toBlock value
   const fromBlock = Math.max(0, options.currentBlock - options.blockHistoryLimit);
+  const toBlock = Math.max(0, options.currentBlock - options.minConfirmations);
```

Comment:
`toBlock` should always be >= `fromBlock`.
```suggestion
  const toBlock = Math.max(fromBlock, options.currentBlock - options.minConfirmations);
```

Reply:
I think there should also be a new unit test that covers this scenario

Reply:
right, i'll add it in